lib/situation_room/check/status_code.ex

```diff
@@ -0,0 +1,14 @@
+defmodule SituationRoom.Check.StatusCode do
+  @moduledoc """
+  Module that checks the status code
+  """
+  @behaviour SituationRoom.Check.Behaviour
+
+  @impl true
+  def run(check) do
```

Comment:
All of these can be pattern matched too so we know what kind of struct we're using